cmd/influxd/generate/command.go

```diff
@@ -0,0 +1,139 @@
+package generate
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/influxdata/influxdb/bolt"
+	"github.com/influxdata/influxdb/cmd/influxd/internal/profile"
+	"github.com/influxdata/influxdb/internal/fs"
+	"github.com/influxdata/influxdb/pkg/data/gen"
+	"github.com/spf13/cobra"
+)
+
+var Command = &cobra.Command{
+	Use:   "generate <schema.toml>",
+	Short: "Generate time series data sets using TOML schema",
+	Long: `
+This command will generate time series data direct to disk using schema 
+defined in a TOML file. Use the help-schema subcommand to produce a TOML 
+file to STDOUT, which includes documentation describing the available options.
+
+
+
+NOTES:
+
+* The influxd server should not be running when using the generate tool
+  as it modifies the index and TSM data.
+* This tool is intended for development and testing purposes only and 
+  SHOULD NOT be run on a production server.
+`,
+	Args: cobra.ExactArgs(1),
+	RunE: generateFE,
+}
+
+var flags struct {
+	printOnly   bool
+	storageSpec StorageSpec
+	profile     profile.Config
+}
+
+func init() {
+	Command.Flags().SortFlags = false
+
+	pfs := Command.PersistentFlags()
+	pfs.SortFlags = false
+	pfs.BoolVar(&flags.printOnly, "print", false, "Print data spec and exit")
```

Comment:
I think in other commands, flag descriptions start lowercase.

Reply:
👍 will fix those – I was using `git --help` a lot, and they had upper case descriptions

Reply:
After a brief discussion and sampling a few tools (`git`, `brew`, `clang`), we'll standardize on sentence casing for flag / option descriptions.